apps/backend/src/routes/signup/onboard/+page.svelte

```diff
@@ -0,0 +1,104 @@
+<!-- JavaScript code for Continue button -->
+<script lang="ts">
+	let confirm: boolean = false;
+</script>
+
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Signup</title>
+</head>
+
+<div class="flex items-center justify-center h-screen">
+	<!-- Signup Form - Logo -->
+
+	<!-- Signup Form - Card -->
+	<div class="w-5/12 p-10 border border-primary rounded-md shadow-lg">
+
+		<!-- Signup Form - Header -->
+		<p class="text-3xl font-semibold text-center mb-5">Get Onboard</p>
```

Comment:
Incorrect usage of html tags. If this is a header, use header tags such as `h1`, `h2`, etc.

Reply:
Changed accordingly, also changed all <!-- Signup Form - ... --> comments to <!-- Onboard Form - ... -->

Reply:
Confirmed